### DISCUSSION TITLE
How is your "discussions board as issue triage" setup going?

### DISCUSSION THREAD

**Discussion by choldgraf** [pradyunsg/furo#250] [Questions] [answered]
This is a meta question for @pradyunsg 🙂

I noticed that this repo has a kind of unconventional setup of asking users to open conversations here, rather than issues directly. I'm curious how this practice is going - i haven't seen it before but it intrigues me as a potential way to boost signal to noise in the issue tracker. Do you find that it makes conversations more productive? Encourages users to help one another out more? Leads to more high quality issues when they're opened?

**Comment by pradyunsg** [comment, marked as answer]
Pretty well IMO.

It lets me enforce forms for the issue tracker without needing to worry about things that don’t fit that mould. I can use the tracker to do exactly that: track actionable things. When things move to the issue tracker, they are usually well defined and clearly actionable.

Support questions organically fall into the discussions, and can be answered by myself or helpful folks around. Plus, support questions and feature requests don’t have to be “closed” once it’s been discussed which… I find nice personally, but you’ll have to ask others about how they feel.

I’ve not had too many conversations here, to judge how it’ll go when off the happy path (think: not-so-great-to-interact-with users and all). OTOH, I’ve definitely had folks who weren’t… uhm… how do I put this nicely… familiar with modern websites, who have been confused by the fact that there’s a new layer to things. That was fine, they seemed corrosive and dropped off, so I guess that’s good.

**Reply by pradyunsg** [reply]
@choldgraf If you wanna know more, I'm happy to chat!

**Comment by CodeCox** [comment]
@choldgraf  FYI

I came across a discussion in another repo where one of the comments point to this '**practice**':
https://github.com/github/feedback/discussions/5709#discussioncomment-1353079

This comment lists 2 podcasts which pertains to the 'interplay' between _Discussions_ & _Issues_  ie. the Maintainer has killed off _Issues_ in favour of _Discusssions_! 

(I've copied the podcast links here for ease of reference)
>   [ I killed the issues tab on the Alpine repo -](https://share.transistor.fm/s/fc094ed5) let's talk about why. (10:14)
>   [ Still Loving GitHub Discussions](https://share.transistor.fm/s/c8c3048a) - Issues are broken. Discussions for LYYFEEEE. (9:39)

Some food for thought?

**Reply by choldgraf** [reply]
Many thanks! I will check it out 🙂
